_includes/javascript.html

```diff
@@ -12,3 +12,22 @@
   ga('create', 'UA-37305346-2', 'auto');
   ga('send', 'pageview');
 </script>
+<script>
+  // This snippet fixes a bug caused by Github's pages-gem using kramdown v1.11.1.
+  // In order for anchor links to point to the correct place in the glossary, they must have an id
+  // This snippet ensures every definition term has an id
+  // See https://github.com/swcarpentry/styles/pull/129
+  $('dt').each(function () {
+    if (!this.id) {
+      var id = $(this).text();
+      // If there's a ( in the name (e.g., "comma-separated values (CSV)") - just take everything up to the first (
```

Comment:
Hmm, the links are inconsistent. On http://swcarpentry.github.io/r-novice-inflammation/reference/#stack-frame, comma separated values points to http://swcarpentry.github.io/r-novice-inflammation/reference/#comma-separated-values-(csv) (comma-separated values (CSV)) but dimensions points to http://swcarpentry.github.io/r-novice-inflammation/reference/#dimensions (dimensions (of an array))